pkg/storaged/stratis-details.jsx

```diff
@@ -599,9 +607,25 @@ export const StratisPoolDetails = ({ client, pool }) => {
 
     const sidebar = <StratisPoolSidebar client={client} pool={pool} />;
 
-    function render_fsys(fsys, offset, total) {
-        const overhead = pool.TotalPhysicalUsed[0] ? (Number(pool.TotalPhysicalUsed[1]) - total) : 0;
-        const pool_total = Number(pool.TotalPhysicalSize) - overhead;
+    const offsets = [];
+    let fsys_total_used = 0;
+    let fsys_total_size = 0;
+    filesystems.forEach(fs => {
+        offsets.push(fsys_total_used);
+        fsys_total_used += fs.Used[0] ? Number(fs.Used[1]) : 0;
+        fsys_total_size += Number(fs.Size);
+    });
+
+    const overhead = pool.TotalPhysicalUsed[0] ? (Number(pool.TotalPhysicalUsed[1]) - fsys_total_used) : 0;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18867-20230814-065829-675f311e-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)